docs/index.rst

```diff
@@ -14,6 +14,7 @@ Welcome to Synfig developers documentation!
    :maxdepth: 2
    :caption: Contents:
 
+   common/Setting up IDE
```

Comment:
I suggest to rename it to "Setting up Netbeans IDE", so we can add more IDE's in the future.

Reply:
Sure